create-envfile.py

```diff
@@ -121,35 +121,21 @@ def _get_vals_to_replace(args):
                 _jsfile = json.load(_json_file)
 
         _vals_to_replace = {
-            key: val for key, val in vars(args).items() if key not in _config
+            key: _jsfile.get(key, val) for key, val in vars(args).items() if key not in _config
         }
+        tcp = 'https' if eval(_jsfile.get("https", f"args.https")) else 'http'
```

Comment:
eval is not safe as a method, use literal_eval if you have to handle strings (even if this should be treated as a boolean)